bitcoin/base58.py

```diff
@@ -91,6 +92,30 @@ def decode(s):
         else: break
     return b'\x00' * pad + res
 
+class InvalidMinikeyError(Base58Error):
+    """Raised for invalid minikeys"""
+    pass
+
+def decode_minikey(minikey):
+    """Decode minikey in str or bytes to standard base58 bytes
+
+    Minikeys are an old key format, for details see
+    https://en.bitcoin.it/wiki/Mini_private_key_format.
+    """
+    if isinstance(minikey, str):
+        minikey = minikey.encode('ascii')
+    length = len(minikey)
+    if length not in [22, 30]:
+        raise InvalidMinikeyError('Minikey length %d is not 22 or 30' % length)
```

Comment:
I suggest using `str("length is {} yo").format(length)` instead of using the `%` operator.